common/source/docs/common-esc-issues.rst

```diff
@@ -85,32 +85,27 @@ will be listed.
 APD ESCs with anti-spark connectors
 -----------------------------------
 
-In December 2022 a serious bug was found in APD F series and HV Pro
-ESCs that impacts vehicles using anti-spark connectors for the
-batteries. The bug can cause the ESC to get into a state where
-throttle can only increase, which will almost always result in a crash
-and can be dangerous for people in the vicinity of the vehicle.
-
-The bug happens due to the following logic in the ESC firmware:
-
-- 200ms after ESC startup the supply voltage is sampled, then multiplied by 1.2 to get a value that is taken as the maximum supply voltage
-- at any time after that if the supply voltage is above this level then an over voltage error is raised
-- while an over voltage error is raised the throttle will not be reduced unless the flight controller demands a zero throttle
-
-The aim of this feature was to prevent motor braking from pushing too
-much power back into the batteries, which could in theory lead to
-overheating of the batteries.
-
-The issue happens if the rise time of the supply voltage is slow
-enough that the voltage does not reach 80% of the true supply voltage
-within 200ms. This can happen when using anti-spark connectors to
-connect the batteries, particularly in combination with long wiring
-and significant levels of added capacitance.
-
-When this happens then on takeoff the flight controller will not be
-able to reduce throttle on affected motors, which will be very likely
-cause the vehicle to crash.
-
-Users of APD ESCs on vehicles using anti-spark connectors should
-contact APD for information on how to upgrade their firmware to
-resolve this bug.
+Vehicles running anti-spark connectors and APD ESCs with firmware versions
+dated before November 2022 may be susceptible to an incorrect setting of
+the over-voltage limit. If incorrectly set (when the pre-charge time constant is over 200 ms),
```

Comment:
I think this is too weak and may lead to users not understanding the serious nature of the risk